code/modules/materials/Mat_Fabrication.dm

```diff
@@ -19,12 +19,6 @@
 	/datum/matfab_recipe/sheet,
 	/datum/matfab_recipe/cell_small,
 	/datum/matfab_recipe/cell_large,
-	/datum/matfab_recipe/simple/insbody,
-	/datum/matfab_recipe/simple/insneck,
-	/datum/matfab_recipe/simple/insmouth,
-	/datum/matfab_recipe/simple/insbell,
-	/datum/matfab_recipe/simple/insbag,
-	/datum/matfab_recipe/simple/insrod,
```

Comment:
you didn't remove the actual datums though?

Reply:
nor the `/obj/item/musicpart`s

Reply:
I accidentally un-did my changes to theses bits of code when undoing some stuff in my end. Whoops. It's already fixed though.